Documentation/network/kubernetes/kubeproxy-free.rst

```diff
@@ -1543,6 +1543,29 @@ For more information, ensure that you have the fix `Pull Request <https://github
     48498    getpeername4
     48494    getpeername6
 
+Known Issues
+############
+
+In case of clusters deployed with cilium version < 1.11.15, there was an issue
+discovered where service backend entries could be leaked in the BPF maps in some. The
+known corner cases that could lead to such leaks are due to race conditions between
```

Comment:
Could you elaborate on those corner cases?

Reply:
The next sentence does elaborate. :)